lib/idp/constants.rb

```diff
@@ -8,6 +8,69 @@ module Vendors
       AAMVA = 'aamva'
     end
 
+    # US State and Territory codes are
+    # taken from the FIPS standard, which
+    # can be found at:
+    # https://www.census.gov/library/reference/code-lists/ansi.html#state
+    STATE_AND_TERRITORY_CODES = %w[
+      AL
+      AK
+      AZ
+      AR
+      CA
+      CO
+      CT
+      DE
+      DC
+      FL
+      GA
+      HI
+      ID
+      IL
+      IN
+      IA
+      KS
+      KY
+      LA
+      ME
+      MD
+      MA
+      MI
+      MN
+      MS
+      MO
+      MT
+      NE
+      NV
+      NH
+      NJ
+      NM
+      NY
+      NC
+      ND
+      OH
+      OK
+      OR
+      PA
+      RI
+      SC
+      SD
+      TN
+      TX
+      UT
+      VT
+      VA
+      WA
+      WV
+      WI
+      WY
+      AS
+      GU
+      MP
+      PR
+      VI
+    ].freeze
```

Comment:
Seems like a good use-case for a [`Set`](https://ruby-doc.org/stdlib-2.7.1/libdoc/set/rdoc/Set.html).

```suggestion
    ].to_set.freeze
```